kossy360/Questioner
# #163243789 Add image support using cloudinary

#### What does this PR do?
- add support for meetup images
- add support for user display picture
- code refactor
- additional unit tests
#### Description of Task to be completed?
- implement addition of images during meetup creation
- implement deletion of images from database and cloudinary after a successful meetup deletion
- implement simultaneously adding and deleting images from database and cloudinary in a single update request
- implement display picture addition on update user route
- implement display picture replacement on update user route
- implement display picture deletion on update user route
#### How should this be manually tested?
Test features using POSTMAN
- send request  ```POST https://k-questioner.herokuapp.com/api/v1/api/v1/auth/signup``` providing an image in the form. The response should contain a displaypicture field containing the url of the image on cloudinary.
- send request  ```PATCH https://k-questioner.herokuapp.com/api/v1/api/v1/users``` providing an image in the form. The response should contain a displaypicture field containing the url of the image on cloudinary. If the meetup previously had a display picture, the same url would still be returned but pointing to the recently uploaded image.
- send request ```POST  https://k-questioner.herokuapp.com/api/v1/api/v1/meetups``` providing images in the form. The response should contain an images field containing an array of the cloudinary urls of all images uploaded
- send request ``` PATCH https://k-questioner.herokuapp.com/api/v1/api/v1/meetups/<meetupId>``` providing images in the form. The response should contain an images field containing an array of the cloudinary urls of all images uploaded
#### Any background context you want to provide?
- All requests containing images should be sent in a form with header ``` content-type: multipart/form-data```
#### What are the relevant pivotal tracker stories?
#163243789
#163389412
#163389536
#163389863
#163391157
#163391931
